domino/routes.py

```diff
@@ -71,9 +71,13 @@ def files_upload(self, path):
     def commits_list(self):
         return self._build_project_url() + "/commits"
 
+    # Deprecated - use blobs_get_v2 instead
     def blobs_get(self, key):
         return self._build_project_url() + "/blobs/" + key
 
+    def blobs_get_v2(self, path, commit_id, project_id):
+        return self.host + f"/api/projects/beta/projects/{project_id}/files/{commit_id}/{path}/content"
```

Comment:
should this be `/v1/` instead of `/beta/`?

Reply:
ahh good catch! yes once I merge in the other PR I will change this to it